src/dstr-binding/default/async-gen-func-named-expr-dflt.template

```diff
@@ -19,7 +19,7 @@ flags: [async]
 
 var callCount = 0;
 var f;
-f = async function* g(/*{ elems }*/ = /*{ vals }*/) {
+f = async function* h(/*{ elems }*/ = /*{ vals }*/) {
```

Comment:
I was asking the rationale for changing the line. Is that the rationale? If so, it would be good to document that in the commit message